### PR TITLE
Feature/fix counter bug

### PR DESCRIPTION
**Pull request by hsleep**
- add TC for exact counter.
- correctly update exact counter if a single row has too many values.
- using the label schema version for counter label.

### CODE REVIEW

**Comment by hsleep** [comment]
@elric-k review plz.

File: s2counter_core/src/main/scala/s2/counter/core/v2/RankingStorageGraph.scala [hunk 1]
```diff
@@ -83,35 +84,39 @@ class RankingStorageGraph(config: Config) extends RankingStorage {
...
-        }
 
-        future
+          future
+        }
+        else {
+          // do nothing
+          Future.successful(false)
+        }
       }
```
Comment (SteamShon) [inline s2counter_core/src/main/scala/s2/counter/core/v2/RankingStorageGraph.scala:120]: above seems just diff from reformat code. right?

Comment (hsleep) [inline s2counter_core/src/main/scala/s2/counter/core/v2/RankingStorageGraph.scala:120]: No, the original code does not check the return value of `checkAndPrepareDimensionBucket`. So, I changed the code to check the return value.

File: s2counter_core/src/main/scala/s2/counter/core/v2/ExactStorageGraph.scala [hunk 2]
```diff
@@ -42,26 +42,58 @@ case class ExactStorageGraph(config: Config) extends ExactStorage {
...
+                ((key, eq), RespGraph(success, result)) <- keyWithEq.zip(respSeq)
+              } yield {
+                (key, (eq, result))
+              }
+            case HttpStatus.SC_BAD_REQUEST =>
+              // logging
+              log.error(s"BAD_REQUEST: $policy $counts")
+              Nil
+            case rc: Int =>
+//              throw new RuntimeException(s"update failed($rc): $policy $counts")
```
Comment (elric-k) [inline s2counter_core/src/main/scala/s2/counter/core/v2/ExactStorageGraph.scala:65]: when update request failed,  it's just logging without exception handling process. 
why did you commented runtime exception block?

Comment (hsleep) [inline s2counter_core/src/main/scala/s2/counter/core/v2/ExactStorageGraph.scala:65]: I want to keep going this spark job although an exception occurred. So, I had changed the code.

**Pull request merged**